Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ghcr.io/osgeo/gdal:ubuntu-small-3.8.0@sha256:3ca7c26ef3aff8c8d134bbaa62e13d
 
 RUN apt-get update
 # Install pip
-RUN apt-get install python3-pip -y
+RUN apt-get install exiftool python3-pip -y
```

Comment:
This is not ideal as you will get random versions of exiftool installed, I am assuming its "fine" as exiftool is everywhere but its not a ideal way of managing packages.

Could we use a python exif variant instead? then it can be version locked in the poetry lock file?

Reply:
Yeah, we have the same problem with `python3-pip`. If anything, we might want to use a Docker image which comes pre-cooked with GDAL, `pip` and Python 3, but I'm not sure that exists.

Reply:
Which this container was made, the python version was very old, I think the gdal containers have much improved and now the python version is pretty new, we could look at removing the pip upgrade. (not sure if it would break anything)

Reply:
Sounds good, but an experiment for a separate PR.